<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1477</width>
      <height>528</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>108</y>
          <width>189</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>249</y>
          <width>189</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>282</y>
          <width>189</width>
          <height>108</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Gamma Transformation</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Scaling Factor:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineGammaScalingFactor"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Gamma:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineGamma"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pbApplyGamma">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>141</y>
          <width>189</width>
          <height>78</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Log Transformation</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Scaling Factor:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineLogScalingFactor"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pbApplyLog">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>11</y>
          <width>132</width>
          <height>48</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Negative Transformation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbApplyNegative">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelOriginal">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
           <property name="text">
            <string>Original</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelOriginalText">
           <property name="text">
            <string>Original</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButtonUndo">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Undo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRedo">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Redo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelResult">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
           <property name="text">
            <string>Result</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelResultText">
           <property name="text">
            <string>Result</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Image"/>
    <addaction name="actionSave_Image"/>
   </widget>
   <widget class="QMenu" name="menuIntensity_Transformation">
    <property name="title">
     <string>Transformations</string>
    </property>
    <addaction name="actionIntensity_Transformation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuIntensity_Transformation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionIntensity_Transformation">
   <property name="text">
    <string>Intensity Transformation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
